### ISSUE TITLE
MH Wilds: Import animations from .motlist files?

### ISSUE THREAD

**Issue by Shadowzj** [NSACloud/RE-Mesh-Editor#5]
In the future will it be possible to import animations from .motlist files?
 And thank you very much. What you guys did was great.

**Comment by NSACloud** [comment]
Yes, I'm going to make a motlist editor at some point. I'm working on a lot of things at the moment so I'm not sure when yet.

**Comment by windshadowruins** [comment]
Thanks for having this in your plan. I can't wait for it 😊. Wilds animation are clean.
